### DISCUSSION TITLE
v3.7.0

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#364] [Announcements]
Thank you to all my supporters. Your support enables me to improve this project and keep pushing forward! 😇

### New Features

- Support fetch lives & other tabs in feed (YouTube)

### Bug Fixes

- Unable to watch new live replays (YouTube)
- Wrong channel names (YouTube)

### Others

- Improve stability for YouTube
- Support new NicoNico (not including NicoLive)

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/v3.7.0'>v3.7.0</a>.</em>

**Comment by ghost** [comment]
MAHORAGA!!!!
HELP
HHEEEEEE
EEEEE
EEEELLPPP
HELP ME
HEEEEELLPPP
HELP MEEEEE
HEEEEELLPPP
HELP MEEEEE
MAHORAGA!!!!

**Reply by vvveeerrr** [reply]
你在说牛魔呢

**Comment by ACS-ak** [comment]
Any plans to add google translate for comments under Bilibili and NicoNico?

**Reply by InfinityLoop1308** [reply]
#438 
But this is a very troublesome feature, involving many issues. Its priority will be relatively low.
